<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionViewer</class>
 <widget class="QDialog" name="TransactionViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lookup Transactions</string>
  </property>
  <widget class="QTableWidget" name="receiptTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>661</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Date [m/d/y]: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewDetailsButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="receiptInformationLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Receipt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="searchByRange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search By Range</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateTwoSearch">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Table</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateSearch</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>viewDetailsButton</tabstop>
  <tabstop>receiptTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
